Directory.Build.targets

```diff
@@ -9,15 +9,6 @@
     <ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>
   </PropertyGroup>
 
-  <!-- temporarily remap AssemblyKey to StrongNameKeyId -->
-  <PropertyGroup>
-    <StrongNameKeyId Condition="'$(AssemblyKey)' == 'MSFT'">Microsoft</StrongNameKeyId>
-    <StrongNameKeyId Condition="'$(AssemblyKey)' == 'ECMA'">ECMA</StrongNameKeyId>
-    <StrongNameKeyId Condition="'$(AssemblyKey)' == 'Open' or '$(IsTestProject)' == 'true'">Open</StrongNameKeyId>
```

Comment:
For test projects we were setting the StrongNameKey to open. Now it will take the one that we default to in the project's Directory.Build.props, is that correct?

Reply:
yes.

Reply:
Well it was supposed to but given we share the library directory props with tests I do need to override it.

Reply:
Yes, that is what I meant, if that was expected by your change or if you needed to keep overriding it.